assets/embedded-files/linkerd/rbac.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: linkerd-installer
+  namespace: linkerd
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: linkerd-installer
+subjects:
+  - kind: ServiceAccount
+    name: linkerd-installer
+    namespace: linkerd
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Do we need `cluster-admin`?

Reply:
Providing fine-grained permissions would be better but we've taken a shortcut here as well: https://github.com/epinio/epinio/blob/main/assets/embedded-files/tekton/admin-role.yaml#L97

We also generally install Epinio as cluster-admins (unless the kubeconfig we use has different permissions).
I see two paths:
1. We assume the cluster "belongs" to Epinio. In this case it's ok to have cluster-admin permissions during Epinio installation (which is the case here and also when we run `epinio install`)
2. We play nice and we use roles with fine grained permissions for each and every action we take: `epinio install`, linkerd install job etc. This may mean we have to split the creation of those roles to a separate step and then use the created roles to actually install Epinio.

Option 2 is a lot more complicated than option 1 and doesn't offer much on clusters where the only tenant is Epinio. I would say let's stick to cluster-admin for installation until we have a use case where this is not ok.

That said, being cluster-admin when doing staging is probably not ok (see the linked file above).

Reply:
okie. We can fix it later.